src/Pact/Native/Internal.hs

```diff
@@ -198,6 +204,26 @@ enforceGuardDef dn =
       _ -> argsError i as
 
 
+enforceVerifierDef :: NativeDef
+enforceVerifierDef = defRNative
+  "enforce-verifier"
+  enforceVerifier
+  (funType tTyBool [("verifiername", tTyString)])
+  [ LitExample $ "(enforce-verifier 'COOLZK)"
+  ]
+  "Enforce that a verifier is in scope."
+  where
+  enforceVerifier :: RNativeFun e
+  enforceVerifier i as = case as of
+    [TLitString verifierName, TList {_tList = (traverse (preview _TLitString) -> Just args) }] -> do
```

Comment:
spurious `do`  (but personally i'd not cram everything into `views` and `>>=` and just add some let bindings)